src/quo/components/navigation/page_nav/view.cljs

```diff
@@ -30,16 +30,17 @@
    & children]
   (into [rn/view {:style (style/container margin-top)}
          (when icon-name
-           [button/button
-            {:type                (button-type background)
-             :icon-only?          true
-             :size                32
-             :on-press            on-press
-             :background          (if behind-overlay?
-                                    :blur
-                                    (when (button-properties/backgrounds background) background))
-             :accessibility-label accessibility-label}
-            icon-name])]
+           [rn/view {:flex-grow 1 :flex-basis 1}
```

Comment:
can you put this in a `:style { }` block?

Reply:
this one is used to align icon within the `container` — adding this code to either container or the button itself leads to completely different